src/frontend/src/views/AgreeTerms/index.tsx

```diff
@@ -120,15 +110,16 @@ export default function AgreeTerms(): JSX.Element {
           >
             Accept
           </Button>
-          <Button
-            disabled={isLoading}
-            color="primary"
-            variant="outlined"
-            isRounded
-            onClick={handleDeclineClick}
-          >
-            Decline
-          </Button>
+          <a href={process.env.API_URL + API.LOG_OUT}>
```

Comment:
does js have a better way of concatenating url paths?  (akin to python's [`posixpath.join`](https://github.com/python/cpython/blob/3.9/Lib/posixpath.py#L71)

Reply:
Ooh interesting! I think FE typically only needs to join two parts, since the latter part is pretty much hard coded all the time, like so 😆 :

```ts
export enum API {
  USER_DATA = "/api/usergroup",
  SAMPLES = "/api/samples",
  LOG_IN = "/login",
  LOG_OUT = "/logout",
  PHYLO_TREES = "/api/phylo_trees",
}
```